### DISCUSSION TITLE
I need to put object design in the video

### DISCUSSION THREAD

**Discussion by pipeddev** [ThibaultBee/StreamPack#153] [Q&A]
Hi everyone, I hope you're fine.
It's possible to add object graphics or design in the live streaming?
for example I need to put my coordinates and minutes, I let an image to understand my requirements.
![image](https://github.com/user-attachments/assets/fdf62f9b-df87-4303-9354-24c0435cffbe)

thanks so much

**Comment by ThibaultBee** [comment]
Hi,
It is not possible yet.
